branca/colormap.py

```diff
@@ -138,18 +142,27 @@ def __call__(self, x):
 
     def _repr_html_(self):
         return (
-            '<svg height="50" width="500">' +
-            ''.join(
-                [('<line x1="{i}" y1="0" x2="{i}" '
-                  'y2="20" style="stroke:{color};stroke-width:3;" />').format(
-                      i=i*1,
-                      color=self.rgb_hex_str(self.vmin +
-                                             (self.vmax-self.vmin)*i/499.))
-                 for i in range(500)]) +
-            '<text x="0" y="35">{}</text>'.format(self.vmin) +
-            '<text x="500" y="35" style="text-anchor:end;">{}</text>'.format(
-                self.vmax) +
-            '</svg>')
+            '<svg height="50" width="500">'
```

Comment:
This is aweful but I suppose that means we should refactor this, not blame Black for it.